samza-core/src/test/java/org/apache/samza/execution/TestJobGraphJsonGenerator.java

```diff
@@ -368,6 +368,31 @@ public void testLegacyTaskApplication() throws Exception {
     assertEquals(0, jsonObject.jobs.size());
   }
 
+  @Test
+  public void testToJobGraphJson() throws Exception {
```

Comment:
Could we also add more tests to test the other changed classes like JobNodeJson? The test here only tests JobGraphJson

Reply:
Thanks for the review @PanTheMan . Had the discussion with @cameronlee314 , we need to figure out one better to way to expose `JobGraphJson`. So I will close this ticket for now.